client/src/app/lib/enums/statDisplaySize.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum StatDisplaySize {
+    Small = "small",
+    Large = "large",
+}
```

Comment:
Sure, let me change it

Reply:
@panteleymonchuk i left utils and constants in camelCase because all the helpers in this project have names in camelCase. Do you think we should also change this?